### ISSUE TITLE
Issue installing strongloop

### ISSUE THREAD

**Issue by astopo** [strongloop/strongloop#320]
I cannot install strongloop globally, because `/usr/local/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng` does not exist. I initially found this issue when trying to deploy to Heroku, using the buildpack [here](https://github.com/strongloop/strongloop-buildpacks), which kept failing because it attempts to install strongloop. Uninstalled strongloop locally on my machine, and tried installing, and I get the same error. Any ideas? This was fine up until sometime at the end of last week.

Output looks like this:

```
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> strong-arc@1.8.8 preinstall /usr/local/lib/node_modules/.staging/strong-arc-e089ca568e2a49f714e3744a579cc8f1
> node .sl-blip.js

/usr/local/bin/slc -> /usr/local/lib/node_modules/strongloop/bin/slc.js
/usr/local/lib
└── (empty)

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "strongloop"
npm ERR! node v5.0.0
npm ERR! npm  v3.5.1
npm ERR! path /usr/local/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/bin/lb-ng'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
```

**Comment by forivall** [comment]
Duplicate of strongloop/loopback#2714, already fixed by #318, just waiting on maintainers to publish a new version to npm.

**Comment by astopo** [comment]
Yay! Thank you

**Comment by davidcheung** [comment]
Thanks for bringing this up, the fix has been published on https://github.com/strongloop/strongloop/commit/979a0458a3d3e7b365ce10e963ac5160629049a8

**Comment by astopo** [comment]
Thanks for the update!
